packages/facility-server/app/routes/apiv1/encounter.js

```diff
@@ -203,7 +203,13 @@ encounterRelations.get(
         {
           model: models.MedicationAdministrationRecord,
           as: 'medicationAdministrationRecords',
-          include: 'reasonNotGiven'
+          include: [
+            'reasonNotGiven', 
+            {
+              association: 'doses',
+              order: [['createdAt', 'ASC']]  // Order doses by oldest first
```

Comment:
this should use `givenTime` I think, `createdAt` is affected by sync so not reliable